server/src/api/observation/observation.controller.ts

```diff
@@ -1,30 +1,65 @@
 import { Request, Response } from 'express';
-import { getManager } from 'typeorm';
 import { ObservationApiModel } from './observation.types';
 import { Observation } from './observation.model';
 import { ApiRequest, EdipiParam } from '../api.router';
+import { Log } from '../../util/log';
+import { ReportSchema } from '../report-schema/report-schema.model';
+import { RosterInfo, getRostersForIndividual } from '../roster/roster.controller';
 import { timestampColumnTransformer } from '../../util/util';
 
+
 class ObservationController {
   async getAllObservations(req: Request, res: Response) {
     return res.json(await Observation.find());
   }
 
   async createObservation(req: ApiRequest<EdipiParam, ObservationApiModel>, res: Response) {
-    return res.json(await createObservation(req));
+
+    Log.info('Creating observation', req.body);
+
+    const reportingGroup = req.body.reportingGroup;
+    const typeId = req.body.typeId;
+
+    const rosters: RosterInfo[] = await getRostersForIndividual(req.body.edipi, req.body.timestamp.toString(10));
```

Comment:
I think this can be simplified a little.

```suggestion
    const rosterInfos = await getRosterInfosForIndividual(edipi, `${timestamp}`);
```

Reply:
done